Aspell Tests Failing On Debian 12

The error I'm running into on Debian 12:
```shell
______________ test_dictionary_formatting[/workspaces/codespell/codespell_lib/tests/../data/dictionary.txt-in_aspell0-in_dictionary0] ______________
codespell_lib/tests/test_dictionary.py:70: in test_dictionary_formatting
    raise AssertionError("\n" + "\n".join(errors))
E   AssertionError: 
E   error 'dateset' should not be in aspell dictionaries (en, en_GB, en_US, en_CA, en_AU) for dictionary /workspaces/codespell/codespell_lib/tests/../data/dictionary.txt
------------------------------------------- generated xml file: /workspaces/codespell/junit-results.xml --------------------------------------------
========================================================== 1 failed, 44 passed in 13.90s ===========================================================
```
On Debian 11, all tests pass.

I suspect this is caused by an update to the `aspell` package from `0.60.8-3` -> `0.60.8-4+b1`.

--- a/codespell_lib/data/dictionary.txt
+++ b/codespell_lib/data/dictionary.txt
@@ -11504,8 +11504,6 @@ datecreatedd->datecreated
 datection->detection
 datections->detections
 datee->date
-dateset->dataset
-datesets->datasets
 datset->dataset
 datsets->datasets
 daty->data, date,

--- a/codespell_lib/data/dictionary_code.txt
+++ b/codespell_lib/data/dictionary_code.txt
@@ -12,6 +12,8 @@ cloneable->clonable
 cmo->com
 copyable->copiable
 creat->create, crate,
+dateset->dataset
+datesets->datasets
 define'd->defined
 deque->dequeue
 doas->does, do as,

